policy/release/tasks.rego

```diff
@@ -385,3 +388,5 @@ _format_missing(o, opt) := desc if {
 } else := sprintf("Required task %q", [o])
 
 _expires_on_annotation := "build.appstudio.redhat.com/expires-on"
+
+_expiry_msg_annotation := "build.appstudio.redhat.com/expiry-message"
```

Comment:
It would be great to add some info on this annotation [here](https://github.com/enterprise-contract/ec-policies/edit/main/antora/docs/modules/ROOT/pages/tasks.adoc).

Reply:
And I reviewed it too :facepalm: